src/gax/src/retry_policy.rs

```diff
@@ -89,63 +89,119 @@ impl RetryFlow {
 ///
 /// Implementations of this trait determine if errors are retryable, and for how
 /// long the retry loop may continue.
-pub trait RetryPolicy: Send + Sync {
+pub trait RetryPolicy: Send + Sync + std::fmt::Debug {
```

Comment:
I wonder if this trait should have a now method for clock manipulation. We could provide a default.

Reply:
I started making this change the the tests looked more complicated.  Maybe I am holding this wrong. We can try again in a future PR.